src/components/quoteBox/quoteBox.tsx

```diff
@@ -47,68 +49,58 @@ const QuoteBox: React.FC<QuoteBoxProp> = ({ quotes, tags }) => {
 
   return (
     <div className="main-container">
-      <div className="display-container p-5">
-        <div className="row filter-container  mb-2 gx-0">
-          <div className="col-12 d-flex justify-content-end align-items-center ">
-            <div className="row ">
-              <div className="col-12 h-100 d-flex justify-content-end align-items-center">
-                <Dropdown onSelect={handleDropdownChange}>
-                  <Dropdown.Toggle variant="light">
-                    {selectedTag[0].toUpperCase() + selectedTag.substring(1)}
-                  </Dropdown.Toggle>
-                  <Dropdown.Menu>
-                    {Array.from(tags).map((tag) => {
-                      return (
-                        <Dropdown.Item eventKey={tag} key={tag}>
-                          {tag[0].toUpperCase() + tag.substring(1)}
-                        </Dropdown.Item>
-                      );
-                    })}
-                  </Dropdown.Menu>
-                </Dropdown>
-              </div>
+        <div className="display-container p-5">
+            <div className="row border-bottom border-dark border-2">
```

Comment:
```suggestion
            <div className="row border-bottom border-dark border-1">
```
Light border is preferred over heavy border, in this case.